{"name": "test_many_case02[case2]", "status": "broken", "statusDetails": {"message": "KeyError: '妹妹'", "trace": "self = <testcase.test_demo.TestDemo object at 0x000001CDCFD9EF70>\ncase = {'弟弟': 12}\n\n    @pytest.mark.parametrize('case',[{'妹妹':18},{'哥哥':25},{'弟弟':12}])\n    def test_many_case02(self,case):\n        print(case,type(case)) # ['妹妹', 18] ['哥哥', 25]\n>       age=case['妹妹']\nE       KeyError: '妹妹'\n\ntest_demo.py:21: KeyError"}, "parameters": [{"name": "case", "value": "{'弟弟': 12}"}], "start": 1650296325400, "stop": 1650296325400, "uuid": "935db028-294d-4246-b942-65c8c72ac780", "historyId": "11d0c6458caa3bb4e4dd758bf71519f0", "testCaseId": "4d1bf863f6e0e91e807ffd73bd9a553d", "fullName": "testcase.test_demo.TestDemo#test_many_case02", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_demo"}, {"name": "subSuite", "value": "TestDemo"}, {"name": "host", "value": "Mikey"}, {"name": "thread", "value": "11344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_demo"}]}